code/modules/atmospherics/machinery/portable/scrubber.dm

```diff
@@ -11,6 +11,7 @@
 	var/volume_rate = 101.325
 	/// Is this scrubber acting on the 3x3 area around it.
 	var/widenet = FALSE
+	resistance_flags = NONE
```

Comment:
Moving this down one line is the only stylistic request I have, LGTM otherwise

Reply:
done